backend/tests/conftest.py

```
@@ -3,6 +3,29 @@
+    if connection:
+        try:
+            cursor = connection.cursor()
+            # Disable foreign key checks to allow truncation
+            cursor.execute("SET FOREIGN_KEY_CHECKS = 0")
+            cursor.execute("TRUNCATE TABLE messages")
+            cursor.execute("TRUNCATE TABLE chats")
+            cursor.execute("TRUNCATE TABLE users")
+            cursor.execute("SET FOREIGN_KEY_CHECKS = 1")
+            connection.commit()
+            cursor.close()
+        except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)